### PR TITLE
OLS-506: Move testing script ot tests

### PR DESCRIPTION
**Pull request by onmete**
## Description

Move validation script to tests. It is only used there. It is causing some problems with imports.

## Type of change

- [x] Refactor

### CODE REVIEW

File: tests/scripts/validate_response.py [hunk 1]
```diff
@@ -5,15 +5,21 @@
...
 from llama_index.embeddings.huggingface import HuggingFaceEmbedding
 from pandas import DataFrame
 from scipy.spatial.distance import cosine, euclidean
 
-sys.path.append(os.path.dirname(os.path.dirname(__file__)))
-from tests.e2e.cluster_utils import create_user, get_user_token, grant_sa_user_access
-from tests.e2e.constants import LLM_REST_API_TIMEOUT
-from tests.e2e.helper_utils import get_http_client
+project_root = Path(__file__).resolve().parent.parent.parent.as_posix()
+sys.path.append(project_root)
```
Comment (bparees) [inline tests/scripts/validate_response.py:15]: is there actually a reason this code has to be run before the imports?  @asamal4 ?

**Review by bparees — COMMENTED** [review]
i'm fine w/ the file move, questions about some of the code itself

File: tests/scripts/validate_response.py (hunk 1 above)
Comment (tisnik) [reply tests/scripts/validate_response.py:15]: IMHO now it is not needed. It was there because of "cross-package" imports (and it's bit dirty hack ;)

**Comment by onmete** [comment]
@asamal4 Path import removed and makefile updated.

**Pull request merged**